src/Std/Data/DTreeMap/Internal/Queries.lean

```diff
@@ -72,12 +72,12 @@ def get? [Ord α] [LawfulEqOrd α] (t : Impl α β) (k : α) : Option (β k) :=
     | .eq => some (cast (congrArg β (compare_eq_iff_eq.mp h).symm) v')
 
 /-- Returns the value for the key `k`. -/
-def get [Ord α] [LawfulEqOrd α] (t : Impl α β) (k : α) (hlk : t.contains k = true) : β k :=
+def get [Ord α] [LawfulEqOrd α] (t : Impl α β) (k : α) (hlk : k ∈ t) : β k :=
   match t with
   | .inner _ k' v' l r =>
     match h : compare k k' with
-    | .lt => get l k (by simpa [contains, h] using hlk)
-    | .gt => get r k (by simpa [contains, h] using hlk)
+    | .lt => get l k (by simpa [instMembershipOfOrd, contains, h] using hlk)
```

Comment:
It would be better to have a lemma that goes with your instance instead of unfolding the instance here.

Reply:
I moved `mem_iff_contains` into `Queries` so that I can use it instead.